go/vt/vtgate/vcursor_impl.go

```diff
@@ -694,6 +694,15 @@ func (vc *vcursorImpl) HasCreatedTempTable() {
 	vc.safeSession.GetOrCreateOptions().HasCreatedTempTables = true
 }
 
+func (vc *vcursorImpl) GetDBDDLPluginName() string {
+	return *dbDDLPlugin
+}
+
+func (vc *vcursorImpl) FindKeyspace(ks string) bool {
+	_, err := vc.topoServer.GetKeyspace(vc.ctx, ks)
+	return err == nil
```

Comment:
Won't this mean that a transient error communicating with topo will make us assume the keyspace doesn't exist? It seems like anything that makes a remote call should return `error` as well to be honest about the possibility that it might fail to get an answer at all. We would only translate the NotFound/NoNode error to `false, nil`.

Reply:
now this is not an issue.